app/views/customer_service/index.html.erb

```diff
@@ -0,0 +1,332 @@
+<%- @page_title = 'Customer Service' -%>
+
+<!-- Don't automatically turn FAX numbers into links -->
+<meta name = "format-detection" content = "telephone=no">
+
+<div class="usa-grid usa-section usa-layout-docs">
+  <%= render 'shared/sidenav' %>
+
+  <div class="usa-width-three-fourths usa-layout-docs-main_content">
+    <h1><%= @page_title %></h1>
+
+    <div class="usa-grid-full customer-service-section" id="local-ppo">
+      <div class="usa-width-one-fourth">
+        <%= image_tag("pppo.png") %>
```

Comment:
For this image (and the others on the page), recommend something like:

```erb
<%= image_tag 'pppo.png', alt: '' %>
```

…and either supplying an empty `alt` attribute value or describing the image. I believe Rails will, by default, set the `alt` attribute to the file name. This will result in some overly-wordy read outs to users of assistive technologies (since the production name for this asset will be something like `pppo-be8f48f5a5178f50a50a8d5d8ed7707d3d7e124e172dfa1d7765f46959a56d37.png`).